Apollo/Binary/Decoder.cs

```diff
@@ -402,7 +402,13 @@ public static class Decoder {
                     reader.ReadBoolean()
                 );
             
-            } else if (t == typeof(KeyFilter)) {
+            } else if (t == typeof(Loop))
```

Comment:
Similarly to Encoder, it'll never know it's a Loop here due to no ID for it existing in Common, so the load would fail